### DISCUSSION TITLE
I have this on centos 7 with kernel 5 stable and with different versions of buildah

### DISCUSSION THREAD

**Discussion by AkakievKD** [containers/buildah#3974] [Q&A]
Running with gitlab-runner 14.10.1 (f761588f)
  on test vHzpKTCM
Preparing the "docker" executor
00:07
Using Docker executor with image quay.io/buildah/stable:v1.[2](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427517244#L2)[3](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427517244#L3).3 ...
Pulling docker image quay.io/buildah/stable:v1.23.3 ...
Using docker image sha256:65868371b67bef5a9016fb5d022ad2bef5e5358[4](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427517244#L4)8e7a0b71b78af8e9[5](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427517244#L5)51881f7 for quay.io/buildah/stable:v1.23.3 with digest quay.io/buildah/stable@sha25[6](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427517244#L6):c83a110f10ad54bd5158490d949ec1fe1f807e56634571629c663be81ccc012d ...
Preparing environment
00:05
Running on runner-vhzpktcm-project-32910647-concurrent-0 via 4fd1db24faf3...
Getting source from Git repository
00:02
Fetching changes with git depth set to 50...
Reinitialized existing Git repository in /builds/my-projects121/frontend-angular/.git/
Checking out ed78fcd6 as main...
Skipping Git submodules setup
Executing "step_script" stage of the job script
00:01
Using docker image sha256:65868371b67bef5a9016fb5d022ad2bef5e535848e7a0b71b78af8e9551881f7 for quay.io/buildah/stable:v1.23.3 with digest quay.io/buildah/stable@sha256:c83a110f10ad54bd5158490d949ec1fe1f807e56634571629c663be81ccc012d ...
$ buildah login -u gitlab+deploy-token-10173[8](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427517244#L8)6 -p DfKoL7k12W6ikmZ-dsZ6 $CI_REGISTRY
Error during unshare(CLONE_NEWUSER): Operation not permitted
time="2022-05-08T0[9](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427517244#L9):44:58Z" level=error msg="error parsing PID \"\": strconv.Atoi: parsing \"\": invalid syntax"
time="2022-05-08T09:44:58Z" level=error msg="(unable to determine exit status)"
Cleaning up project directory and file based variables
00:01
ERROR: Job failed: exit code 1

**Comment by AkakievKD** [comment]
Running with gitlab-runner 14.10.1 (f761588f)
  on test vHzpKTCM
Preparing the "docker" executor
00:04
Using Docker executor with image quay.io/buildah/stable:v1.[2](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427523467#L2)1.0 ...
Pulling docker image quay.io/buildah/stable:v1.21.0 ...
Using docker image sha256:2f[3](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427523467#L3)9d89eb3[4](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427523467#L4)2e2fc[5](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427523467#L5)ec3e833b[6](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427523467#L6)3dd986ee5e387b165de263171cebe05493b2f9 for quay.io/buildah/stable:v1.21.0 with digest quay.io/buildah/stable@sha256:3cc22eeb8eae9c0244d19d9fd7b081ca116aad0bfc51dfa57e57e305afd25f7a ...
Preparing environment
00:00
Running on runner-vhzpktcm-project-32910647-concurrent-0 via 4fd1db24faf3...
Getting source from Git repository
00:02
Fetching changes with git depth set to 50...
Reinitialized existing Git repository in /builds/my-projects121/frontend-angular/.git/
Checking out 779f765f as main...
Skipping Git submodules setup
Executing "step_script" stage of the job script
00:02
Using docker image sha256:2f39d89eb342e2fc5ec3e833b63dd986ee5e387b165de263171cebe05493b2f9 for quay.io/buildah/stable:v1.21.0 with digest quay.io/buildah/stable@sha256:3cc22eeb8eae9c0244d19d9fd7b081ca116aad0bfc51dfa57e57e305afd25f7a ...
$ buildah login -u gitlab+deploy-token-10173[8](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427523467#L8)6 -p DfKoL7k12W6ikmZ-dsZ6 $CI_REGISTRY
Login Succeeded!
$ buildah bud -t "$CI_REGISTRY_IMAGE:$CI_COMMIT_TAG" myNewApp/Dockerfile
Error during unshare(CLONE_NEWUSER): Operation not permitted
time="2022-05-08T0[9](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427523467#L9):51:57Z" level=error msg="error parsing PID \"\": strconv.Atoi: parsing \"\": invalid syntax"
time="[20](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427523467#L20)[22](https://gitlab.com/my-projects121/frontend-angular/-/jobs/2427523467#L22)-05-08T09:51:57Z" level=error msg="(unable to determine exit status)"
Cleaning up project directory and file based variables
00:01
ERROR: Job failed: exit code 1

**Comment by AkakievKD** [comment]
My runner in docker with latest version
How fix it?

**Comment by rhatdan** [comment]
This looks like Docker is using seccomp filtering and it is blocking unshare syscall.
